sites/partners/page_content/locale_overrides/general.json

```diff
@@ -221,29 +221,33 @@
   "listings.lottery.duplicateStringPlural": "%{sets} unresolved duplicate sets.",
   "listings.lottery.duplicatesConfirm": "You should resolve any duplicates before running the lottery.",
   "listings.lottery.export": "Export lottery data",
-  "listings.lottery.exportContent": "This file includes the lottery raw rank and preferences data for all applications.",
-  "listings.lottery.exportContentNoPreferences": "This file includes the lottery raw rank for all applications.",
   "listings.lottery.exportContentTimestamp": "This data was generated from the lottery that was run on %{date} at %{time}.",
-  "listings.lottery.exportFile": "File includes randomized general pool and preference data.",
-  "listings.lottery.exportFileNoPreferences": "File includes randomized general pool data.",
+  "listings.lottery.exportFile": "This file includes the lottery raw rank and preferences data for all applications.",
+  "listings.lottery.exportFileNoPreferences": "This file includes the lottery raw rank for all applications.",
   "listings.lottery.history": "History",
   "listings.lottery.noData": "No lottery data",
   "listings.lottery.noDataDescription": "It looks like you haven't run a lottery for this listing yet.",
-  "listings.lottery.noDataDescriptionPartner": "Doorway staff will run the lottery and perform a quality check on the results before making the lottery results accessible to you on %{date}. Before viewing the lottery results, you will be prompted to publish the lottery data, which will email results to applicants. Please email partners@housingbayarea.org with any questions.",
+  "listings.lottery.noDataDescriptionPartner": "Admins will run the lottery and perform a quality check on the results before making the lottery results accessible to you on the scheduled lottery date. Before viewing the lottery results, you will be prompted to publish the lottery data, which will email results to applicants. Please email",
```

Comment:
This doesn't match what is stated in `listings.lottery.publishContent` about what is emailed to applicants

Reply:
It's the confirmed copy in the doc after the latest round of changes, but I hear ya